CharlieBackend.Api/Controllers/HomeworksController.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using CharlieBackend.Core;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using CharlieBackend.Core.DTO.Homework;
+using Microsoft.AspNetCore.Authorization;
+using Swashbuckle.AspNetCore.Annotations;
+using CharlieBackend.Business.Services.Interfaces;
+
+namespace CharlieBackend.Api.Controllers
+{
+    /// <summary>
+    /// Controller to make operations with homework
+    /// </summary>
+    [Route("api/homeworks")]
+    [ApiController]
+    public class HomeworksController : ControllerBase
+    {
+        private readonly IHomeworkService _homeworkService;
+
+        /// <summary>
+        /// Homework controllers constructor
+        /// </summary>
+        public HomeworksController(IHomeworkService homeworkService)
+        {
+            _homeworkService = homeworkService;
+        }
+
+        /// <summary>
+        /// Adds homework
+        /// </summary>
+        [SwaggerResponse(200, type: typeof(HomeworkDto))]
+        [Authorize(Roles = "Admin, Mentor")]
+        [HttpPost]
+        public async Task<ActionResult> PostHomework([FromBody]CreateHomeworkDto request)
+        {
+            var results = await _homeworkService
+                        .CreateHomeworkAsync(request);
+
+            return results.ToActionResult();
+        }
+
+        /// <summary>
+        /// Gets all homeworks of course
+        /// </summary>
+        [SwaggerResponse(200, type: typeof(List<HomeworkDto>))]
+        [Authorize(Roles = "Admin, Mentor")]
+        [HttpGet("/api/courses/{id}/homeworks")]
+        public async Task<ActionResult> GetHomeworksOfCourse(long courseId)
```

Comment:
![image](https://user-images.githubusercontent.com/5232107/102896258-c2c95d80-446e-11eb-9109-c13e935966d2.png)
please name action parameter the same as in route to avoid having two parameters in swagger